app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt

```diff
@@ -77,7 +77,7 @@ class DragDropTestActivityTest {
       scenario.onActivity { activity ->
         attachDragDropToActivity(activity)
       }
-      onView(withId(R.id.drag_drop_recycler_view)).perform(
+      onView(withId(recyclerViewId = R.id.drag_drop_recycler_view)).perform(
```

Comment:
Do not add named arguments to `withId`.
Update this to `onView(withId(R.id.drag_drop_recycler_view))`

Reply:
This is the reason why tests are failing too.

Reply:
@Naman-1234 PTAL

Reply:
Ok Sir Working on it.